### PR TITLE
Upgrade: Update v1.1.1 known issues

### PR DESCRIPTION
**Pull request by bk201**
Update v1.1.1
- Support upgrade matrix.
- Known issues.
- Add simplified Chinese translation.

### CODE REVIEW

File: docs/upgrade/v1-0-3-to-v1-1-1.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+
+- Related issue:
+  - [[BUG] Upgrade stop at upgrading node3 stage, stuck in Pre-drained status](https://github.com/harvester/harvester/issues/3021)
+- Workaround:
+  - https://github.com/harvester/harvester/issues/3021#issuecomment-1288747614
+
+---
+
+### 5. Additional certificates are gone after an upgrade
```
Comment (bk201) [inline docs/upgrade/v1-0-3-to-v1-1-1.md:86]: @FrankYang0529 Please help check this session; thanks!

File: docs/upgrade/v1-0-3-to-v1-1-1.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+After upgrading from v1.0.3, the file `/etc/pki/trust/anchors/additional-ca.pem` configured via the [`additioanl-ca` setting](https://docs.harvesterhci.io/v1.1/advanced/settings#additional-ca) is gone.
+
+To fix this, the user needs to access the Settings page (Harvester GUI, `Advanced -> Settings`): 
+- Edit the `additional-ca` setting. Back up the current value first and click `Save`.
```
Comment (FrankYang0529) [inline docs/upgrade/v1-0-3-to-v1-1-1.md:91]: We may need to backup the current value, clean it up, and then `Save`, so we don't hit [the annotation check](https://github.com/harvester/harvester/blob/0cdb5a5b1cb104c825eddb88002e81648ff422dd/pkg/controller/master/setting/handler.go#L77-L79).

**Review by FrankYang0529 — CHANGES_REQUESTED** [review]

File: docs/upgrade/v1-0-3-to-v1-1-1.md (hunk 2 above)
Comment (bk201) [reply docs/upgrade/v1-0-3-to-v1-1-1.md]: Thanks, and add the "clear value" statement.

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade/v1-0-3-to-v1-1-1.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  - https://github.com/harvester/harvester/issues/3021#issuecomment-1288747614
+
+---
+
+### 5. Additional certificates are gone after an upgrade
+
+After upgrading from v1.0.3, the file `/etc/pki/trust/anchors/additional-ca.pem` configured via the [`additioanl-ca` setting](https://docs.harvesterhci.io/v1.1/advanced/settings#additional-ca) is gone.
+
+To fix this, the user needs to access the Settings page (Harvester GUI, `Advanced -> Settings`): 
+- Edit the `additional-ca` setting. Back up the current value first and click `Save`.
```
Comment (FrankYang0529) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade/v1-0-3-to-v1-1-1.md:91]: It looks like we didn't update here.

**Review by FrankYang0529 — CHANGES_REQUESTED** [review]

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade/v1-0-3-to-v1-1-1.md (hunk 3 above)
Comment (bk201) [reply i18n/zh/docusaurus-plugin-content-docs/current/upgrade/v1-0-3-to-v1-1-1.md]: Nice catch!

File: docs/upgrade/v1-0-3-to-v1-1-1.md (hunk 1 above)
Comment (FrankYang0529) [reply docs/upgrade/v1-0-3-to-v1-1-1.md:86]: This part looks good to me. Thanks.

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade/v1-0-3-to-v1-1-1.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+---
+
+### 2. 升级卡在预清空节点状态 (情况 2)
+
+升级卡在下图所示的状态：
+
+![](/img/v1.1/upgrade/known_issues/3216-stuck-pre-drain.png)
+
+使用者观察到同时有多个节点被禁止排程：
```
Comment (guangbochen) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade/v1-0-3-to-v1-1-1.md:39]:
```suggestion
使用者同时观察到有多个节点被禁止调度：
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade/v1-0-3-to-v1-1-1.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+- 相关 issue：
+  - [[BUG] Upgrade stop at upgrading node3 stage, stuck in Pre-drained status](https://github.com/harvester/harvester/issues/3021)
+- 解决方法：
+  - https://github.com/harvester/harvester/issues/3021#issuecomment-1288747614
+
+---
+
+### 5. 其他受信 CA 证书在更新后遗失
+
+由 v1.0.3 版本升级后，借由设置 [`additioanl-ca`](https://docs.harvesterhci.io/v1.1/advanced/settings#additional-ca) 产生的档案 `/etc/pki/trust/anchors/additional-ca.pem` 遗失。
```
Comment (guangbochen) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade/v1-0-3-to-v1-1-1.md:87]:
```suggestion
从 v1.0.3 升级后，通过 [`additioanl-ca`](https://docs.harvesterhci.io/v1.1/advanced/settings#additional-ca) 设置配置的 `/etc/pki/trust/anchors/additional-ca.pem` 消失了。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade/v1-0-3-to-v1-1-1.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+解决办法为存取设置页面（由Harvester图形介面，高级选项 -> 设置）：
```
Comment (guangbochen) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade/v1-0-3-to-v1-1-1.md:89]:
```suggestion
要解决此问题，用户需要访问设置页面（Harvester GUI，Advanced -> Settings）：
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade/v1-0-3-to-v1-1-1.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+- 编辑 `additional-ca` 设置。备份目前的值后，清空值，点击保存。
```
Comment (guangbochen) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade/v1-0-3-to-v1-1-1.md:90]:
```suggestion
- 编辑 `additional-ca` 设置。 先备份当前值，清除当前值，点击保存。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade/v1-0-3-to-v1-1-1.md (hunk 4 above)
Comment (bk201) [reply i18n/zh/docusaurus-plugin-content-docs/current/upgrade/v1-0-3-to-v1-1-1.md]: Thanks for the suggestions.

File: docs/upgrade/v1-0-3-to-v1-1-1.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+- Related issue:
+  - [[BUG] Multiple nodes pre-drains in an upgrade](https://github.com/harvester/harvester/issues/3216)
+- Workaround:
+  - https://github.com/harvester/harvester/issues/3216#issuecomment-1328607004
+
+---
+
+### 3. Upgrade stuck in upgrading first node: Job was active longer than specified deadline
+
+An upgrade fail, as shown in the screenshot below:
```
Comment (vickyhella) [inline docs/upgrade/v1-0-3-to-v1-1-1.md:59]:
```suggestion
An upgrade fails, as shown in the screenshot below:
```

File: docs/upgrade/v1-0-3-to-v1-1-1.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+```
+
+- Related issue:
+  - [[BUG] Multiple nodes pre-drains in an upgrade](https://github.com/harvester/harvester/issues/3216)
+- Workaround:
+  - https://github.com/harvester/harvester/issues/3216#issuecomment-1328607004
+
+---
+
+### 3. Upgrade stuck in upgrading first node: Job was active longer than specified deadline
```
Comment (vickyhella) [inline docs/upgrade/v1-0-3-to-v1-1-1.md:57]:
```suggestion
### 3. An upgrade is stuck in upgrading the first node: Job was active longer than the specified deadline
```

Comment (vickyhella) [reply docs/upgrade/v1-0-3-to-v1-1-1.md:57]: I changed this sentence only to keep the structure the same as other counterparts~

File: docs/upgrade/v1-0-3-to-v1-1-1.md [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+- Related issue:
+  - [[BUG] Upgrade stop at upgrading node3 stage, stuck in Pre-drained status](https://github.com/harvester/harvester/issues/3021)
+- Workaround:
+  - https://github.com/harvester/harvester/issues/3021#issuecomment-1288747614
+
+---
+
+### 5. Additional certificates are gone after an upgrade
+
+After upgrading from v1.0.3, the file `/etc/pki/trust/anchors/additional-ca.pem` configured via the [`additioanl-ca` setting](https://docs.harvesterhci.io/v1.1/advanced/settings#additional-ca) is gone.
```
Comment (vickyhella) [inline docs/upgrade/v1-0-3-to-v1-1-1.md:88]:
```suggestion
After upgrading from v1.0.3, the file `/etc/pki/trust/anchors/additional-ca.pem` configured via the [`additioanl-ca` setting](https://docs.harvesterhci.io/v1.1/advanced/settings#additional-ca) disappears.
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade/v1-0-3-to-v1-1-1.md [hunk 11]
```diff
@@ -0,0 +1,93 @@
+---
+sidebar_position: 2
+sidebar_label: 升级到 v1.1.1
+title: "从 v1.0.3/v1.1.0 升级到 v1.1.1"
```
Comment (vickyhella) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade/v1-0-3-to-v1-1-1.md:4]:
```suggestion
sidebar_label: 从 v1.0.3/v1.1.0 升级到 v1.1.1
title: "从 v1.0.3/v1.1.0 升级到 v1.1.1"
```

File: docs/upgrade/v1-0-3-to-v1-1-1.md [hunk 12]
```diff
@@ -0,0 +1,94 @@
+---
+sidebar_position: 2
+sidebar_label: Upgrade to v1.1.1
+title: "Upgrade from v1.0.3/v1.1.0 to v1.1.1"
```
[the remark this answers is not in the document]
Comment (bk201) [reply docs/upgrade/v1-0-3-to-v1-1-1.md]: @vickyhella, thanks. Commit all suggestions.

**Review by vickyhella — APPROVED** [review]

**Review by guangbochen — APPROVED** [review]
LGTM, and please help to squash it for merge, thanks.

**Pull request merged**